modules/virt-checking_status_of_node_maintenance_cr_tasks.adoc

```diff
@@ -41,7 +41,7 @@ items:
     - httpd
     - httpd-manual
     phase: Running
-  lastError: "Last failure message"   <2>
+    lastError: "Last failure message" <2>
```

Comment:
@aireilly, do you know anyone in QE that can verify this change? Thanks!

Reply:
@slintes can you comment/lgtm on this please - @jboxman is fixing YAML typos in the OCP docs with this PR. Jason's change looks correct to me.

Reply:
I'm not QE, but lgtm 👍🏼

Reply:
@slintes oops - who can QE?

Reply:
/lgtm